### PR TITLE
add path instructions for windows

### PR DESCRIPTION
**Pull request by theadrianbao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,10 +112,18 @@ of random tech startup business names.
...
    actual OpenAI API key.
 
+    On macOS or Linux systems,
+
     ```bash
     export OPENAI_API_KEY=$YOUR_OPENAI_API_KEY
     ```
 
+    On Windows systems,
+
```
Comment (zetianluo) [inline README.md:122]: Thank you for adding the powershell command! Could you add the terminal command as well for windows user?

Comment (zetianluo) [reply README.md:122]: I believe is 'set OPENAI_API_KEY=your_openai_api_key'

Comment (theadrianbao) [reply README.md:122]: setx is persistent and should work for both cmd and powershell. This should be fine. https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/setx

**Review by zetianluo — APPROVED** [review]
Looks good to me, thank you!

**Pull request merged**